sci-mathematics/octave/octave-5.1.0.recipe

```diff
@@ -23,16 +28,23 @@ PROVIDES="
 	cmd:octave_cli_$portVersion$secondaryArchSuffix = $portVersion
 	cmd:octave_config$secondaryArchSuffix = $portVersion
 	cmd:octave_config_$portVersion$secondaryArchSuffix = $portVersion
-	lib:liboctave$secondaryArchSuffix = $portVersion
-	lib:liboctinterp$secondaryArchSuffix = $portVersion
+	lib:liboctave$secondaryArchSuffix = $liboctaveVersionCompat
+	lib:liboctinterp$secondaryArchSuffix = $liboctinterpVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	cmd:gnuplot$secondaryArchSuffix
```

Comment:
There is no $secondaryArchSuffix for gnuplot
https://github.com/haikuports/haikuports/blob/b82dc69cb6d8cb3e8bb4f327473950587c8190bb/sci-visualization/gnuplot/gnuplot-5.2.5.recipe#L25

Reply:
thx